PRIVACY.md

```diff
@@ -0,0 +1,47 @@
+* What choices are available to YOU regarding the use of your data.
+* The security procedures in place to protect the misuse of information YOU provide.
+* How YOU can correct any inaccuracies in the information.
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, YOU refers to a website that has installed Flagrow Bazaar and its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
+
+### Information Collection, Use, and Sharing
+WE are the sole owners of the information collected by the APPLICATION during the course of normal operation. 
+
+If YOU choose to connect the APPLICATION to the WEBSITE, then WE also own any information provided. Unless YOU ask us not to, we may contact YOU via email in the future to tell you about specials, new products or services, or changes to this privacy policy.
+
+If YOU choose to email us or directly contact us in another way, WE also have access to/collect information that YOU voluntarily provide. WE will also use this information to respond to YOU, regarding the reason YOU contacted us. WE will not share your information with any third party outside of our organization, other than as necessary to fulfill your request, e.g. to provide a service.
+
+**WE will not sell or rent this information to anyone for any purposes.**
+
+### What We Collect
+
+To provide the best service for this APPLICATION, WE may collect the following from YOU. 
+
+* The contents of the Flarum configuration file where the APPLICATION is installed.
+* The contents of files relating to Composer configuration where the APPLICATION is installed.
+* Preferences marked in the APPLICATION, such as favorites.
+* The IP and url where the APPLICATION is installed.
+* An email address YOU provide  (if connected to the WEBSITE).
+* A name YOU provide (if connected to the WEBSITE).
+* Information that connects YOU to the Flarum instance where the APPLICATION is installed.
+* Other statistical information related to the Flarum instance where the APPLICATION is installed. 
+
+### Your Access to and Control Over Information
+YOU may opt out of any future contacts from us at any time. YOU can do the following at any time by contacting us via the email address given on our WEBSITE at flagrow.io:
+
+* See what data we have about YOU, if any.
+* Change/correct any data we have about YOU.
+* Have us delete any data we have about YOU.
+* Express any concern YOU have about our use of data we store on YOU.
+
+### Security
+WE take precautions to protect information about YOU. When YOU submit sensitive information via the WEBSITE, the information is protected both online and offline.
+
+Wherever WE collect sensitive information (such as an email address), that information is encrypted and transmitted to us in a secure way. YOU can verify this by looking for a lock icon in the address bar and looking for "https" at the beginning of the address of the Web page.
```

Comment:
Maybe "whenever" instead of "wherever" ? Not sure

We could mention that all data transmission is secured by TLS, instead of specifically mentioning the green lock.

For instance if users install Bazaar on an unsecured (http) host, they won't see a lock, but the requests made by the backend to flagrow.io will still be secured by TLS